vsphere_plugin_common/__init__.py

```diff
@@ -1465,6 +1468,48 @@ def _add_network(self, network, datacenter):
             guest_map.adapter.subnetMask = str(nw.netmask)
         return nicspec, guest_map
 
+    def _get_nic_keys_for_remove(self, server):
+        # get nics keys in tempalte before our changes,
+        # should be used keys instead mac, as macs will be changed after
```

Comment:
This would be better phrased as 'we must use keys instead of mac addresses, as macs will be (etc)